### PR TITLE
'has roost' change dc:creator to dcterms:contributor

### PR DESCRIPTION
**Pull request by wdduncan**

### CODE REVIEW

**Comment by wdduncan** [comment]
@matentzn @cthoyt @anitacaron the qc is failing b/c of non-conforming values for the `created_by` annotation.  
E.g., here is a list produced after running `make test`.

```
FAIL Rule ../sparql/term-editor-uri-violation.sparql: 33 violation(s)
subject,orcid
http://purl.obolibrary.org/obo/RO_0002207,has developmental precursor
http://purl.obolibrary.org/obo/valid_for_go_gp2term,pg
http://purl.obolibrary.org/obo/valid_for_go_annotation_extension,pg
http://purl.obolibrary.org/obo/RO_0002020,dos
http://purl.obolibrary.org/obo/RO_0012002,pg
http://purl.obolibrary.org/obo/RO_0002021,dos
http://purl.obolibrary.org/obo/RO_0012012,pg
http://purl.obolibrary.org/obo/RO_0002023,dos
http://purl.obolibrary.org/obo/RO_0002019,dos
http://purl.obolibrary.org/obo/RO_0012004,pg
http://purl.obolibrary.org/obo/RO_0002026,dos
http://purl.obolibrary.org/obo/RO_0012001,pg
http://purl.obolibrary.org/obo/RO_0012005,pg
http://purl.obolibrary.org/obo/RO_0002029,dos
http://purl.obolibrary.org/obo/RO_0002017,dos
http://purl.obolibrary.org/obo/RO_0002015,dos
http://purl.obolibrary.org/obo/RO_0012006,pg
http://purl.obolibrary.org/obo/RO_0002018,dos
http://purl.obolibrary.org/obo/RO_0012003,pg
http://purl.obolibrary.org/obo/RO_0002022,dos
http://purl.obolibrary.org/obo/valid_for_go_ontology,pg
http://purl.obolibrary.org/obo/RO_0012007,pg
http://purl.obolibrary.org/obo/RO_0012008,pg
http://purl.obolibrary.org/obo/RO_0002024,dos
http://purl.obolibrary.org/obo/RO_0002025,dos
http://purl.obolibrary.org/obo/RO_0002013,dos
http://purl.obolibrary.org/obo/RO_0012000,pg
http://purl.obolibrary.org/obo/RO_0012009,pg
http://purl.obolibrary.org/obo/RO_0002014,dos
http://purl.obolibrary.org/obo/RO_0012010,pg
http://purl.obolibrary.org/obo/valid_for_gocam,pg
http://purl.obolibrary.org/obo/RO_0012011,pg
http://purl.obolibrary.org/obo/RO_0002016,dos
```

File: src/ontology/ro-edit.owl
Comment (balhoff) [inline src/ontology/ro-edit.owl:6434]: Why was the xsd:dateTime removed?

File: src/ontology/ro-edit.owl
Comment (balhoff) [inline src/ontology/ro-edit.owl:6424]: Why was the xsd:dateTime removed?

File: src/ontology/ro-edit.owl
Comment (balhoff) [inline src/ontology/ro-edit.owl:6416]: Why was the xsd:dateTime removed?

**Review by balhoff — CHANGES_REQUESTED** [review]
It looks like some `xsd:dateTime` datatypes were lost. Also a bunch of `xsd:string`s, but that doesn't matter—was it on purpose?

How does this relate to 'has roost'? It seems like a general cleanup.

**Comment by wdduncan** [comment]
@balhoff when I changed the annotation property I didn't specify the datatype. I assumed the datatype would carry over to the new annotation. My mistake :(

This PR originated from my oversight of merging [NTR: has roost](https://github.com/oborel/obo-relations/pull/672) prior to the qc checks finishing :(

File: src/ontology/ro-edit.owl
Comment (wdduncan) [reply src/ontology/ro-edit.owl]: added xsd:dateTime

File: src/ontology/ro-edit.owl
Comment (wdduncan) [reply src/ontology/ro-edit.owl]: added xsd:dateTime

File: src/ontology/ro-edit.owl
Comment (ddooley) [reply src/ontology/ro-edit.owl]: This might be connected: Something to be on the watch for - Protege 5.6.1 seems to have a bug in it such that if you edit an annotation, the form it pops up shows "xsd:string" as default datatype even if existing content has another datatype.  So editing an existing time entry - even just looking at one and saving it, causes the datatype to be overwritten (/cleared out). See https://github.com/protegeproject/protege/issues/1152

**Comment by gouttegd** [comment]
@wdduncan Mind if I commit directly to this PR to fix the remaining QC issues?

**Comment by wdduncan** [comment]
@gouttegd I don't mind at all!  
Thanks!

**Comment by gouttegd** [comment]
Done and the CI checks now pass.

**Comment by cthoyt** [comment]
Please please please do not merge this until the commit history on the main branch is fixed

**Comment by wdduncan** [comment]
What needs to be fixed? I'm not following.

**Comment by gouttegd** [comment]
@cthoyt The point of this PR is precisely to fix the breakage that was introduced by the merging of #672.

I agree that a hard reset to cancel #672 would be cleaner, but not now that the merge has occurred on the public master branch. Hard resets are good on a local repository to cancel commits that have never been published, but once they have been pushed on a public repo, it’s too late. Rewriting history on a public branch is a bad idea.

**Comment by cthoyt** [comment]
please see https://github.com/oborel/obo-relations/issues/732. @gouttegd either way, can you help remove all of the spurious diff in this PR? like tons of changes are totally irrelevant to the content update

**Comment by gouttegd** [comment]
@cthoyt Those “spurious changes” are _not_ irrelevant: they are here to fix the real spurious changes that were introduced in #672. Obviously, when you introduced a spurious change, the fix to it _also_ looks like a spurious change.

Note that if you go the #732 route and reset the main branch to before #672 was merged, then this PR becomes irrelevant, so I don’t know why your last item in the description of #732 is to ”fix #731”.

(I am personally opposed to any form of history re-writing on a public repo. That being said, I don’t know what is the OBO policy on the matter – if even there is one –, so if nobody else has a problem with that, I won’t object ~~too much~~. All I want is to fix the master branch so that I can then update and merge #709.)

**Comment by wdduncan** [comment]
> All I want is to fix the master branch so that I can then update and merge https://github.com/oborel/obo-relations/pull/709.

That is what is most important. Github purity is secondary to do this.

**Comment by gouttegd** [comment]
> This only happened a day ago and I don't think that changing it will affect anyone who isn't already involved in PRs on this repo anyway

#672 was merged _3 days_ ago. Hard-resetting the master branch to before it was merged will affect anyone who has updated their local copy since then. That’s reason enough for not doing so, in my opinion.

**Review by balhoff — APPROVED** [review]

**Comment by wdduncan** [comment]
Thanks for your help @balhoff and @gouttegd !

File: src/ontology/ro-edit.owl
Comment (anitacaron) [inline src/ontology/ro-edit.owl:49]: Still, have to redo some changes! No CARO is imported anymore.

**Pull request merged**
